src/main/java/uk/gov/justice/digital/delius/controller/secure/AuthenticationController.java

```diff
@@ -42,7 +40,7 @@
                     @ApiResponse(code = 401, message = "Unauthorised", response = ErrorResponse.class)
             })
     @PostMapping("/authenticate")
-    public ResponseEntity authenticate(@NotNull @Valid @ApiParam(value = "Authentication Details", required = true) @RequestBody final AuthUser authUser) {
+    public ResponseEntity<Void> authenticate(@NotNull @Valid @ApiParam(value = "Authentication Details", required = true) @RequestBody final AuthUser authUser) {
```

Comment:
I was on the fence at one point, but I prefer the global exception handlers instead these days rather than having to return a response entity here.  Not that I'd change it, more as a comment :grin:

Reply:
Yep, agreed, no use exception strategy